src/helpers/buildTransactionsSectionsSelector.js

```diff
@@ -45,12 +50,18 @@ const addContactInfo = contacts => txn => {
 };
 
 const buildTransactionsSections = (
+  accountAddress,
+  accountAssetsData,
   contacts,
   requests,
+  theme,
   transactions,
   isFocused,
-  initialized
+  initialized,
+  navigate
 ) => {
+  const onTransactionPress = transactionPressBuilder(navigate);
```

Comment:
this function have new reference every time this whole sections builder is executed